### ISSUE TITLE
Bro

### ISSUE THREAD

**Issue by Oluwamaya** [iSammyk/Investment#1]
How do we handle this

**Comment by iSammyk** [comment]
which of the probems are you asking about

**Comment by Oluwamaya** [comment]
I have fixed it just work on the dashboard olodo lol..

**Comment by Oluwamaya** [comment]
In be like nah that pull request dey give person badge abi ..

**Comment by iSammyk** [comment]
yes i think so, but this one too suppose get hin badge. and follow back abeg, make i go tell gc people self
